portal-web/test/functional/com/liferay/portalweb/macros/Page.macro

```diff
@@ -625,9 +628,14 @@
 			<var name="privacy" value="${privacy}" />
 		</execute>
 
-		<execute function="AssertElementPresent" locator1="SitePages#LIST_GROUP_ITEM">
-			<var name="key_itemName" value="${pageName}" />
-		</execute>
+		<if>
+			<isset var="depth" />
+			<then>
+				<execute macro="SitePages#expandPageTree" />
```

Comment:
Please define a var for each variable that gets passed into the expandPageTree macro. Example:
`<var name ="depth" value="${depth}" />`